source/main/modules/window-manager/index.ts

```diff
@@ -806,7 +879,7 @@ export default class WindowManager {
     * Show the dialog for choosing a directory
     * @return {string[]} An array containing all selected paths.
     */
-  async askDir (win?: BrowserWindow|null): Promise<string[]> {
+  async askDir (win?: BrowserWindow | null): Promise<string[]> {
```

Comment:
Don't make whitespace changes to code unrelated to the pull request. I don't think the Zettlr developers will approve the whitespace changes.